### DISCUSSION TITLE
Behaviour like ctrl+n

### DISCUSSION THREAD

**Discussion by jan0sch** [hrsh7th/nvim-cmp#1199] [Q&A] [answered]
Hi,

is it possible to have behaviour like the ctrl+n completion from nvim?

I tried setting `['<C-n>'] = cmp.mapping.complete()` and disabled automatic starting of autocomplete but the thing I miss is that in plain [n]nvim the ctrl+n pops up the completion _and_ also jumps to the next item i.e. you're able to scoll through all available options without changing to a different key/shortcut.
With cmp the ctrl+n only starts the popup but I have to switch to using the cursor keys for selecting the appropriate entry.

Kind regards,

Jens

**Comment by fitrh** [comment, marked as answer]
You can use `cmp.visible()`

```lua
["<C-n>"] = cmp.mapping({
    i = function()
        if cmp.visible() then -- pop-up menu is visible
            cmp.select_next_item()
        else
            cmp.complete() -- open the pop-up menu
        end
    end,
})
```

**Reply by jan0sch** [reply]
Thanks a lot! :heart:
